src/API/Actions/AbstractAction.php

```diff
@@ -0,0 +1,64 @@
+<?php declare(strict_types=1);
```

Comment:
Our coding standard forces the declare(strict_types=1); not in a newline? @michz @solutionDrive-Alt